packages/actions/src/helpers/s3.ts

```diff
@@ -0,0 +1,320 @@
+import { Functions, httpsCallable, HttpsCallable } from "firebase/functions"
+import mime from "mime-types"
+import fs from 'fs'
+import { ETagWithPartNumber, ChunkWithUrl } from "../../types"
+import fetch from "@adobe/node-fetch-retry"
+import https from "https"
+import dotenv from "dotenv"
+
+dotenv.config()
```

Comment:
Minor: To avoid confusion as we are using `env.test` in the `actions` package, change to ```dotenv.config({ path: `${__dirname}/../../.env.test` })```. Please, check if nothing breaks with this change!